lib/tests/packages_detect.rb

```diff
@@ -193,6 +217,11 @@ def detect_formulae!(args:)
     modified_formulae #{modified_formulae.join(" ").presence  || "(none)"}
     deleted_formulae  #{@deleted_formulae.join(" ").presence  || "(none)"}
     formulae_to_fetch #{@formulae_to_fetch.join(" ").presence || "(none)"}
+
+    testing_casks  #{@testing_casks.join(" ").presence  || "(none)"}
```

Comment:
Feels like this should only be printed on `args.only_casks_detect?` (and not print formulae above in that case).